driver_cpl/cime_config/namelist_definition_drv.xml

```diff
@@ -223,30 +223,6 @@
     </values>
   </entry>
 
-  <entry id="atm_adiabatic">
-    <type>logical</type>
-    <category>expdef</category>
-    <group>seq_infodata_inparm</group>
-    <desc>
-      true => turn on adiabatic mode in cam
-    </desc>
-    <values>
-      <value>.false.</value>
-    </values>
-  </entry>
-
-  <entry id="atm_ideal_phys">
-    <type>logical</type>
-    <category>expdef</category>
-    <group>seq_infodata_inparm</group>
-    <desc>
-      true => turn on ideal physics mode in cam
-    </desc>
-    <values>
-      <value>.false.</value>
-    </values>
-  </entry>
-
   <entry id="aqua_planet">
```

Comment:
CAM no longer looks at these since it is not the driver should not be involved in internal atmosphere model settings. 
This is also already fixed in the ACME atmosphere model.